corporate/lib/stripe.py

```diff
@@ -1170,6 +1189,50 @@ def set_required_plan_tier(self, required_plan_tier: int) -> str:
             plan_tier_name = CustomerPlan.name_from_tier(new_plan_tier)
         return f"Required plan tier for {self.billing_entity_display_name} set to {plan_tier_name}."
 
+    def create_fixed_price_plan(self, fixed_price: int) -> str:
+        customer = self.get_customer()
+        if customer is None:
+            customer = self.update_or_create_customer()
+
+        if customer.required_plan_tier is None:
+            raise SupportRequestError("Required plan tier should not be set to None")
+        required_plan_tier_name = CustomerPlan.name_from_tier(customer.required_plan_tier)
+
+        fixed_price_cents = fixed_price * 100
+        plan = get_current_plan_by_customer(customer)
+        if plan is not None:
+            fixed_price_plan = CustomerPlan.objects.create(
+                customer=customer,
+                automanage_licenses=True,
```

Comment:
I'm not sure what this value should be ... but does it get overwritten anyway?

It might be best to have some comments blocking off which values we expect to be used vs. not.